src/components/Content/Pursuance/PursuanceMenuItem.js

```diff
@@ -9,10 +9,15 @@ const PursuanceMenuItem = ({ pursuanceId, label, action, icon, location, classNa
       className={"menu-item " + className}
       to={`${rootPath}/${action}`}
       isActive={
-        () =>
-          ((action === 'tasks' && (location.pathname === rootPath || location.pathname === rootPath + '/'))
-          ||
-          location.pathname.indexOf(`/${action}`) !== -1)
+        () => (
+          (action === 'tasks' && (
+            location.pathname === rootPath 
+            || 
+            location.pathname === rootPath + '/' 
+            || 
+            location.pathname.indexOf(`/${action}`) !== -1
+          ))
+        )
```

Comment:
There is also more to this function now in `develop`; it treats the `discuss` action differently, which is what we want.